From 5be39a8c30fba163b3f8358c1b7db02facecb538 Mon Sep 17 00:00:00 2001
From: Laszlo Ersek <lersek@redhat.com>
Date: Sat, 11 Jan 2014 17:59:51 +0100
Subject: [PATCH 01/22] Partially revert "rhel: Drop "cfi.pflash01" and "isa-ide" device"

RH-Author: Laszlo Ersek <lersek@redhat.com>
Message-id: <1389463208-6278-2-git-send-email-lersek@redhat.com>
Patchwork-id: 56615
O-Subject: [RHEL-7.0 qemu-kvm PATCH 01/18] Partially revert "rhel: Drop "cfi.pflash01" and "isa-ide" device"
Bugzilla: 1032346
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>
RH-Acked-by: Amos Kong <akong@redhat.com>
RH-Acked-by: Andrew Jones <drjones@redhat.com>
RH-Acked-by: Kevin Wolf <kwolf@redhat.com>

This reverts the cfi.pflash01 elimination in
commit f6852e27575e095269a47f434d318b5fc8822e0a.

Signed-off-by: Laszlo Ersek <lersek@redhat.com>
---
 default-configs/x86_64-softmmu.mak | 1 +
 1 file changed, 1 insertion(+)

Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 default-configs/x86_64-softmmu.mak |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/default-configs/x86_64-softmmu.mak b/default-configs/x86_64-softmmu.mak
index e7c48ec..7b8b05c 100644
--- a/default-configs/x86_64-softmmu.mak
+++ b/default-configs/x86_64-softmmu.mak
@@ -19,6 +19,7 @@ CONFIG_I8257=y
 CONFIG_IDE_PIIX=y
 CONFIG_PIIX_PCI=y
 CONFIG_I8259=y
+CONFIG_PFLASH_CFI01=y
 CONFIG_TPM_TIS=$(CONFIG_TPM)
 CONFIG_PCI_HOTPLUG=y
 CONFIG_MC146818RTC=y
-- 
1.7.1

